README.md

```diff
@@ -95,7 +95,13 @@ Veracode recommends that you use the toplevel parameter if you want to ensure th
 
 ### `deleteIncompleteScan`
 
-**Optional** BOOLEAN - Set to true to automatically delete the current scan if there are any errors when uploading files or starting the scan. If the include or exclude parameters are set, this parameter deletes the scan if there are errors when starting the scan after module selection. Defaults to false.
+**Optional** Boolean - Set to true to automatically delete the current scan if there are any errors when uploading files or starting the scan. If the include or exclude parameters are set, this parameter deletes the scan if there are errors when starting the scan after module selection. Defaults to false.
+
+**In Java API Wrapper version >=22.5.10.0 this parameter has changed to an Integer. One of these values:**
```

Comment:
```suggestion
**Optional**

**In Java API Wrapper version >=22.5.10.0 this parameter has changed to an Integer. One of these values:**
```

Reply:
@mhowes-veracode - I meant this line. Sorry for being ambiguous

Reply:
No problem. Wasn't sure if you were referring to this repo or the Veracode Docs. Thanks!